src/pages/home.vue

```diff
@@ -1,177 +1,55 @@
-                <q-btn flat icon="close" @click="basicModal2 = false">
-                  <div style="margin-left: 10px; text-transform: initial;">Bring on the fruit</div>
-                </q-btn>
-              </div>
-              <img src="~assets/header-pop.jpg" alt="ww" style="width: 683px;">
-              <q-btn flat icon="close" class="absolute-top-left" @click="basicModal2 = false" style="color: white;" />
-              <div style="padding: 50px">
-                <div class="q-display-1 q-mb-md">Basic Modal</div>
-                <p v-for="n in 15" :key="`a-${n}`">Scroll down to close</p>
-                <q-btn to="/orders" class="Order" rounded>
-                  <div>Order</div>
-                </q-btn>
-              </div>
-            </q-modal-layout>
-          </q-modal>
-          <!-- popup modal -->
-        </div>
-      </q-carousel-slide>
-    </q-carousel>
-    <q-carousel class="component-top" style="max-width: 700px;margin:auto; height:300px">
-      <q-carousel-slide img-src="~assets/panel.jpg">
-      </q-carousel-slide>
-    </q-carousel>
-    <q-carousel>
-      <q-modal v-model="opened">
-        <h4>Basic Modal</h4>
-        <q-btn color="primary" @click="opened = false" label="Close" />
-      </q-modal>
-    </q-carousel>
+    <news v-for="(item,index) in news" :key="`news${index}`" :item="item">
+      <news-details :item="item"></news-details>
+    </news>
   </div>
 </template>
-
-<style>
-.transbox {
-  list-style-type: none;
-  padding: 0;
-  overflow: hidden;
-  background-color: #fff;
-  position: -webkit-sticky; /* Safari */
-  position: sticky;
-  top: 0;
-  border-left: 1px solid #d9d2d2;
-  border-right: 1px solid #d9d2d2;
-  opacity: 0.01;
-  filter: alpha(opacity=60); /* For IE8 and earlier */
-  margin-bottom: -38px;
-}
-
-.custom-caption-top {
-  text-align: center;
-  color: white;
-  margin-top: 20px;
-  margin-left: 20px;
-}
-.custom-caption-mid {
-  text-align: center;
-  color: white;
-  margin-top: 50px;
-  margin-left: 20px;
-}
-.component-top {
-  padding: 5px 0 10px 0;
-}
-.btnlearn {
-  color: black;
-  background-color: white;
-  margin-bottom: 20px;
-  margin-left: 20px;
-}
-.q-display-1 {
-  font-size: 40px;
-}
-.inner-top {
-  font-size: 24px;
-  color: cyan;
-}
-.q-layout-header {
-  box-shadow: inherit;
-}
-.Order {
-  background-color: rgb(49, 138, 49);
-}
-.q-modal-layout-content {
-  overflow-x: hidden;
-}
-</style>
-
 <script>
+import news from '../components/news'
+import newsDetails from '../components/newsDetails'
 export default {
+  name: 'PageIndex',
+  components: {
+    news,
+    newsDetails,
+  },
   data() {
     return {
-      opened: false,
-      basicModal: false,
-      basicModal2: false,
-      imageSrc: '~assets/component.jpg',
-      types: [
+      news: [
         {
+          img: 'assets/component.jpg',
           label: 'Learn More',
-          show: () => {
-            this.basicModal = true
-          },
+          isBtnHidden: false,
```

Comment:
swap properties' name and order to: (for all objects)
          img:
          isBtnHidden: 
          label:
          title:
          subTitle: 
          titleContent: 
          content: